### PR TITLE
Issue #6207: Added Xpath IT regression test for SuperClone

### PR DESCRIPTION
**Pull request by Rohanraj123**
part of #6207 

Documentation : [SuperClone](https://checkstyle.sourceforge.io/checks/coding/superclone.html#SuperClone)

### CODE REVIEW

File: src/it/resources/org/checkstyle/suppressionxpathfilter/superclone/SuppressionXpathRegressionSuperCloneInnerClone.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package org.checkstyle.suppressionxpathfilter.superclone;
+
+public class SuppressionXpathRegressionSuperCloneInnerClone {
+    public Object clone() // warn
+    {
```
Comment (romani) [inline src/it/resources/org/checkstyle/suppressionxpathfilter/superclone/SuppressionXpathRegressionSuperCloneInnerClone.java:5]: This is same as SuppressionXpathRegressionSuperClonePlainAndSubclasses
Make this method to non clone to get violation in inner class

**Review by romani — CHANGES_REQUESTED** [review]
Items

**Review by romani — APPROVED** [review]
Ok to merge, thanks a lot for contribution

**Comment by romani** [comment]
Please fix CI failure https://app.circleci.com/pipelines/github/checkstyle/checkstyle/21996/workflows/846277a2-affd-4d24-944c-e86adc2f457b/jobs/436325/artifacts

**Comment by Rohanraj123** [comment]
> Please fix CI failure https://app.circleci.com/pipelines/github/checkstyle/checkstyle/21996/workflows/846277a2-affd-4d24-944c-e86adc2f457b/jobs/436325/artifacts

done !

**Pull request merged**